### ISSUE TITLE
Show results of people you've gotten wrong with more frequency

### ISSUE THREAD

**Issue by botwhytho** [jasonaowen/recurse-faces#28]
Reinforce the learning of people for whom you don't know the names well. Feedback on this would be appreciated

**Comment by jasonaowen** [comment]
This is a good idea!

It implies keeping track of successful and unsuccessful guesses (is that related to #23?), and who you've seen. Should that be in local storage, or on the server? Local storage is all client-side, which means per-browser; server-side needs API + database, and might run into Heroku's PostgreSQL free-tier 10,000 rows cap, but will persist longer and across browsers.

**Comment by botwhytho** [comment]
Yes, this would build on top of #23, and potentially #24 (although graphing/displaying any of this data is not a prerequisite for this). As this feature does not require central storage, just passing extra parameters on the GET request (assuming this would be something like the IDs to exclude in SQL queries), I would think that a local-storage-type solution might work here. I also would like to learn how to use local-storage, because I haven't before :smile: 

Is there a way to track 'dependencies' of issues to other issues on Github? Milestones? Potentially projects? I feel like I might need this to understand what I or somebody else can work on next. I know you can reference other issues but this doesn't seem to provide the view I'm looking for

**Comment by brainwane** [comment]
Spaced repetition would be great to help reinforce/improve learning!

I personally would be fine with `localstorage` for this even if that means it would not persist across browsers and might not persist across sessions/longer durations.

@botwhytho this project's maintainers haven't created any projects or milestones to use in arranging issues, but even if they had, I think those do not provide the kind of thing I think you're asking about (a way to indicate that issue A depends on issue B as a prerequisite).
